app/mailers/rdv_mailer.rb

```diff
@@ -22,6 +22,13 @@ def cancel_rdv_coronavirus(rdv, user)
     mail(to: user.email, subject: "RDV annulé le #{l(rdv.starts_at, format: :human)} avec #{rdv.organisation.name}")
   end
 
+  def cancellation(rdv, user)
+    @rdv = rdv
+    @user = user
+    @lieu = @rdv.motif.public_office? ? Lieu.find_by(address: @rdv.location) : nil
```

Comment:
- you can use `@rdv.public_office?` (I think, or else you should delegate the method in the model)
- it'd be better to add a `lieu` method in the Rdv that returns what you just wrote
```
def lieu
  return nil unless public_office?

  Lieu.find_by(address: location)
end
```